packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -201,6 +202,7 @@ class Auth {
         const response = await this._api.shopperLogin.getAccessToken(options)
         // Check for error response before handling the token
         if (response.status_code) {
```

Comment:
Let's add a comment here that clarifies the expectation that any `status_code` at all is a signal of error (because success omits `status_code`

Reply:
A comment has been added.

Also, after doing some testing, only SLAS getAccessToken will return a `status_code` that is a string.
The other endpoints where an auth related HTTPError can be produced have a `status` code that is a number.
So rather than change HTTPError to cast the status to a number, we set a number here.